tests/test_postprocess.py

```diff
@@ -0,0 +1,306 @@
+"""Unit tests dedicated to predicted label postprocessing
+"""
+
+import numpy as np
+import pytest
+
+from deeposlandia import postprocess
+
+
+def test_get_image_paths(tanzania_image_size):
+    """Test the image path getting function
+
+    Preprocessed image filenames must end with ".png"
+    """
+    filenames = postprocess.get_image_paths(
+        "./tests/data", "tanzania", tanzania_image_size, "grid_066"
+    )
+    assert np.all([f.endswith(".png") for f in filenames])
+
+
+def test_extract_images(tanzania_image_size, tanzania_nb_output_testing_images):
+    """Test the image extraction function, that retrieve the accurate data in a
+    'numpy.array' starting from a list of filenames
+
+    This image data must be shaped as (nb_filenames, image_size, image_size, 3).
+    """
+    filenames = postprocess.get_image_paths(
+        "./tests/data", "tanzania", tanzania_image_size, "tanzania_sample"
+    )
+    images = postprocess.extract_images(filenames)
+    assert len(images.shape) == 4
+    assert images.shape[0] == tanzania_nb_output_testing_images
+    assert images.shape[1] == tanzania_image_size
+    assert images.shape[2] == tanzania_image_size
+    assert images.shape[3] == 3
+
+
+def test_get_labels(tanzania_image_size, tanzania_nb_labels):
+    """Test the label retrieving from dataset glossary
+    """
+    labels = postprocess.get_labels(
+        "./tests/data", "tanzania", tanzania_image_size
+    )
+    assert len(labels) == tanzania_nb_labels
+    with pytest.raises(ValueError):
+        postprocess.get_labels(
+            "./elsewhere-on-the-disk", "wrong-dataset", tanzania_image_size
+        )
+
+
+def test_get_trained_model(tanzania_image_size, tanzania_nb_labels):
+    """Test the semantic segmentation model retrieving
+
+    Postprocessing implies getting a 'unet' model (i.e. segmentation
+    segmentation), hence the model must be shaped as follows:
```

Comment:
typo. you wrote "segmentation segmentation" :slightly_smiling_face:

Reply:
A new AI model, you don't know it?
...
Modified.